rls/src/actions/mod.rs

```diff
@@ -425,6 +435,24 @@ impl InitActionContext {
     }
 }
 
+/// Read edition from the manifest
```

Comment:
nit:
```suggestion
/// Read package edition from the Cargo manifest
```
Since crate targets can have different editions associated with them